libobs-d3d11/d3d11-subsystem.hpp

```diff
@@ -1002,8 +1002,15 @@ struct gs_device {
 	ComPtr<ID3D11Device> device;
 	ComPtr<ID3D11DeviceContext> context;
 	uint32_t adpIdx = 0;
+	string Name;
+	size_t DedicatedVideoMemory;
+	size_t SharedSystemMemory;
```

Comment:
These don't appear to be used for anything?

Reply:
I think the were a holdover from previous experimentation. I'm indifferent if we keep or remove them.

Reply:
Removed.